cyences_app_for_splunk/default/macros.conf

```diff
@@ -1232,6 +1232,9 @@ iseval = 0
 definition = search *
 iseval = 0
 
+[cs_windows_defender_malware_detected_local_time_converter]
+definition = eval Detection_Time = strftime(strptime(Detection_Time, "%Y-%m-%dT%H:%M:%S.%QZ"), "%F %T %Z")
```

Comment:
Make this macro generic by adding 2 params:: field name and format of the timestamp

definition = eval "$fieldname$" = strftime(strptime('$fieldname$', "$timeformat$"), "%F %T %Z")

Macro name should be something like:: cs_local_time_conversion

Reply:
`cs_local_time_conversion(Detection_Time, "%Y-%m-%dT%H:%M:%S.%QZ")`